### ISSUE TITLE
(Android) Improve TalkBack experience

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#2]
Includes:

- Announce the current selection state of a card.
- Announce the index of a card in a dealt card pile and the count of cards in the pile.
- Prevent (if possible) TalkBack from announcing unhelpful data when at a card.

**Comment by gbarkerz** [comment]
In my testing, TalkBack didn't announce the selection state or the "M of N" for items in CollectionViews. So for now (from V1.14) the names of the dealt card explicitly includes the selection state and the "M of N". This is not what I'd want to have to do, but at least makes the app usable until it's understood why TalkBack doesn't announce the expected data.

No improvements have been made to reduce the amount of unwanted data announced by TalkBack which is out of the control of the app.
